### PR TITLE
T18657

### PR DESCRIPTION
**Pull request by smspillaz**
https://phabricator.endlessm.com/T18657

### CODE REVIEW

**Review by joaquimrocha — DISMISSED** [review]
See my comments in the Phabricator task.

File: data/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,3 @@
+iconoverridesdir = $(datadir)/eos-application-tools/icon-overrides
+
+iconoverrides_DATA = eos-icon-overrides.ini
```
Comment (cosimoc) [inline data/Makefile.am]: I think this should be `dist_iconoverrides_DATA`, or the file be added to `EXTRA_DIST`

File: data/eos-icon-overrides.ini [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[Overrides]
+org.videolan.VLC.desktop=eos-vlc.desktop
+com.microsoft.Skype.desktop=eos-skype.desktop
+com.spotify.Client.desktop=eos-spotify.desktop
```
Comment (cosimoc) [inline data/eos-icon-overrides.ini]: Please keep this list alphabetical

**Review by cosimoc — CHANGES_REQUESTED** [review]

**Review by cosimoc — APPROVED** [review]
This one LGTM now

**Pull request merged**